src/Sql/Sql/TransparentDataEncryption/Cmdlet/RevalidateAzureRmSqlManagedInstanceTransparentDataEncryptionProtector.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Model;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Cmdlet
+{
+    /// <summary>
+    /// Defines the Revalidate-AzSqlInstanceTransparentDataEncryptionProtector cmdlet
+    /// </summary>
+    [Cmdlet("Invoke", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceTransparentDataEncryptionProtectorRevalidation", SupportsShouldProcess = true, DefaultParameterSetName = DefaultParameterSet)]
+    [OutputType(typeof(AzureRmSqlManagedInstanceTransparentDataEncryptionProtectorModel))]
+    public class RevalidateAzureRmSqlManagedInstanceTransparentDataEncryptionProtector : AzureSqlRmManagedInstanceTransparentDataEncryptionProtectorBase
+    {
+        /// <summary>
+        /// Gets or sets whether or not to run this cmdlet in the background as a job
+        /// </summary>
```

Comment:
Add PassThru to keep consistent with invoke-AzSqlDatabaseTransparentDataEncryptionProtectorRevalidation